template/src/store/reducers/user.ts

```diff
@@ -1,7 +1,7 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Because in reducers, we will defined a lot of empty function (usually for API call), so I have to disable this rule here in these files, or there will be a lot of error from ESLint

Reply:
Remove it. I don't think we need that empty function

Reply:
I updated. Remove all unnecessary functions